pkg/cmd/start/start_pipeline.go

```diff
@@ -202,37 +201,59 @@ func (o *StartPipelineOptions) createMetaPipeline(jobname string) error {
 
 	jxClient, ns, err := o.JXClientAndDevNamespace()
 	if err != nil {
-		return errors2.Wrap(err, "failed to create JX client")
+		return errors.Wrap(err, "failed to create JX client")
 	}
 
 	sr, err := kube.FindSourceRepository(jxClient, ns, owner, repo)
 	if err != nil {
-		return errors2.Wrap(err, "cannot determine git source URL")
+		return errors.Wrap(err, "cannot determine git source URL")
 	}
 
 	sourceURL, err := kube.GetRepositoryGitURL(sr)
 	if err != nil {
-		return errors2.Wrapf(err, "cannot generate the git URL from SourceRepository %s", sr.Name)
+		return errors.Wrapf(err, "cannot generate the git URL from SourceRepository %s", sr.Name)
 	}
 	if sourceURL == "" {
 		return fmt.Errorf("no git URL returned from SourceRepository %s", sr.Name)
 	}
 
-	po := create.StepCreatePipelineOptions{
-		SourceURL:    sourceURL,
-		Job:          jobname,
-		PullRefs:     pullRefs,
-		Context:      o.Context,
-		CustomLabels: o.CustomLabels,
-		CustomEnvs:   o.CustomEnvs,
+	log.Logger().Info("creating meta pipeline client")
```

Comment:
actually, here is should be Debug, since we are in a user facing command and info logging is equal to user feedback